### ISSUE TITLE
[Bug]: Corpse cleanup function apparently getting double-called

### ISSUE THREAD

**Issue by jaj22** [official-antistasi-community/A3-Antistasi#1302]
*Version:* Unstable

*Mods:* Probably any

*Environment*: MP host and MP dedi at least

***Issue:***
The postmortem calls tend to look like this, sometimes with the null call before the non-null one:
```
21:10:56 "1480.36: [Antistasi] | DEBUG | fn_postmortem | PostMortem Called"
21:10:56 "1480.36: [Antistasi] | DEBUG | fn_postmortem | Pausing for 60 minutes before cleaning victim: O Alpha 1-4:1 and group: O Alpha 1-4"
21:10:56 "1480.36: [Antistasi] | DEBUG | fn_postmortem | PostMortem Called"
21:10:56 "1480.36: [Antistasi] | ERROR | fn_postmortem | Function failed called with null param."
```
Probably something dumb going on that should be fixed.

**Comment by jaj22** [comment]
Looks like the Killed event handlers are often called a second time with all-null parameters for no clear reason. occupantInvaderUnitKilledEH doesn't detect this case and some logic is still applied with null parameters: This postmortem call, which is harmless, but also changing city support. So this should be fixed. Other killed EHs may also need parameter checks.

**Comment by jaj22** [comment]
Arma bug, fixed somewhere around 2.06
